### PR TITLE
New package: Samsung.EasySoftwareManager version 1.2.22.16

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

### CODE REVIEW

File: manifests/s/Samsung/EasySoftwareManager/1.2.22.16/Samsung.EasySoftwareManager.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.EasySoftwareManager
+PackageVersion: 1.2.22.16
+Installers:
+- InstallerUrl: https://org.downloadcenter.samsung.com/downloadfile/ContentsFile.aspx?CDSite=US&CttFileID=5009450&CDCttType=SW&ModelType=C&ModelName=DP300A2A-A01US&VPath=SW/201209/20120926134248625/Easy_Software_Manager_1.2.22.16.ZIP
```
Comment (Trenly) [inline manifests/s/Samsung/EasySoftwareManager/1.2.22.16/Samsung.EasySoftwareManager.installer.yaml:7]:
```suggestion
- InstallerUrl: https://downloadcenter.samsung.com/content/SW/201209/20120926134248625/Easy_Software_Manager_1.2.22.16.ZIP
```

File: manifests/s/Samsung/EasySoftwareManager/1.2.22.16/Samsung.EasySoftwareManager.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.EasySoftwareManager
+PackageVersion: 1.2.22.16
+PackageLocale: en-US
+Publisher: Samsung
+PackageName: Samsung Easy Software Manager
```
Comment (Trenly) [inline manifests/s/Samsung/EasySoftwareManager/1.2.22.16/Samsung.EasySoftwareManager.locale.en-US.yaml:8]:
```suggestion
PackageName: Easy Software Manager
```

File: manifests/s/Samsung/EasySoftwareManager/1.2.22.16/Samsung.EasySoftwareManager.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.EasySoftwareManager
+PackageVersion: 1.2.22.16
+PackageLocale: en-US
+Publisher: Samsung
```
Comment (Trenly) [inline manifests/s/Samsung/EasySoftwareManager/1.2.22.16/Samsung.EasySoftwareManager.locale.en-US.yaml:7]:
```suggestion
Publisher: Samsung Electronics CO., LTD.
```

**Review by Trenly — CHANGES_REQUESTED** [review]
image

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Comment by upintheairsheep** [comment]
> Automatic Validation ended with:
> image

Appears to be an interactive only installer

**Pull request closed without merging**